lib/google_drive/file.rb

```diff
@@ -213,9 +213,9 @@ def rename(title)
     alias title= rename
 
     # Creates copy of this file with the given title.
-    def copy(title)
+    def copy(title, params = {})
```

Comment:
Thanks! Looks useful. But it's a bit confusing to call it `params`. In this library, `params` usually refer to the third argument of copy_file e.g., `fields` and `supports_team_drives`. How about calling it `file_properties` instead?

Reply:
I am totally fine with that! Renaming!